tests/testNRyield.cc

```diff
@@ -90,9 +103,13 @@ int main(int argc, char* argv[]) {
   if (argc < 4) {
     G4cerr << "Usage: testNRyield Emin Emax unit [points=100] [Material=G4_Si]\n\n"
 	   << "Arguments: Emin, Emax: specify energy range to test\n"
+       << "Optional: [klow] [khigh] [Elow] [Ehigh] [useEnergyDependentK=1]\n"
 	   << "           units: Valid Geant4 unit string (e.g., 'MeV')\n"
 	   << "           points: Number of energy points to sample\n"
 	   << "           material: Valid Geant4 NIST name; default is G4_Si\n"
+       << "           klow, khigh: k parameter range\n"
```

Comment:
Add a note that these five parameters only apply to, and are passed to, the EmpiricalNIEL model, not the other models.